luigi/contrib/external_program.py

```diff
@@ -121,7 +122,7 @@ def _clean_output_file(self, file_object):
     def run(self):
         args = list(map(str, self.program_args()))
 
-        logger.info('Running command: %s', ' '.join(args))
+        logger.info('Running command: %s', ' '.join(six.shlex_quote(arg) for arg in args))
```

Comment:
this needs to be `six.moves.shlex_quote` i think.

Reply:
It seems that you were right.